src/test/data/JsonSerializableTaskBookTest/typicalTasksTaskBook.json

```diff
@@ -5,18 +5,21 @@
     "date" : "09-10-2022",
     "startTime" : "09:00",
     "endTime" : "10:00",
-    "tagged" : [ "friends", "colleagues" ]
+    "tagged" : [ "friends", "colleagues" ],
+    "persons" : ["Alice Pauline", "Benson Meier"]
   }, {
     "name" : "Training",
     "date" : "29-02-2020",
     "startTime" : "14:00",
     "endTime" : "16:00",
-    "tagged" : [ "colleagues" ]
+    "tagged" : [ "colleagues" ],
+    "persons" : ["Carl Kurz"]
   }, {
     "name" : "Shareholder Meeting",
     "date" : "29-02-2020",
     "startTime" : "14:00",
     "endTime" : "16:00",
-    "tagged" : [ "colleagues" ]
+    "tagged" : [ "colleagues" ],
+    "persons" : ["Carl Kurz", "Benson Meier"]
```

Comment:
If task will be implemented to ensure person is not attached to a conflicting task (as per User Story given), make sure to change typical tasks in memory to avoid conflicting events in sample TaskBook.